scripts/update-readme.sh

```diff
@@ -7,7 +7,8 @@ CMD="$1"
 
 TMP=`mktemp`
 TMP2=`mktemp`
-./terraform-provider-snowflake -doc > "$TMP"
+VERSION="$(cat VERSION)"
+./terraform-provider-snowflake_"$VERSION" -doc > "$TMP"
```

Comment:
maybe BASE_BINARY_NAME is better, that way we're not duplicating how we assemble the name of the binary here and the makefile